src/transactions/TransactionFrame.cpp

```diff
@@ -59,6 +59,35 @@ namespace
 // roughly 112 million lumens.
 int64_t const MAX_RESOURCE_FEE = 1LL << 50;
 
+uint32_t
+getNumDiskReadEntries(SorobanResources const& resources,
+                      SorobanTransactionData::_ext_t const& ext)
+{
+    // First count classic entry reads
+    uint32_t count = 0;
+    auto countClassic = [&count](auto keys) {
```

Comment:
Should this be `auto const&`? I'm not sure if this might end up copying the footprint...